### DISCUSSION TITLE
4.3.0-rc.2

### DISCUSSION THREAD

**Discussion by ismcagdas** [abpframework/abp#8544] [Releases]
4.3.0-rc.2

<hr /><em>This discussion was created from the release <a href='https://github.com/abpframework/abp/releases/tag/4.3.0-rc.2'>4.3.0-rc.2</a>.</em>

**Comment by alexandis** [comment]
Hi. When are you going to publish 4.3 final so we were able to update to it?

**Reply by maliming** [reply]
Today 💯

**Reply by ebicoglu** [reply]
completing in 20 mins
